envs/snakemake-workflows.yaml

```diff
@@ -5,14 +5,15 @@ channels:
   - defaults
 dependencies:
   - pkgs/main::python=3.6
-  - bioconda::snakemake>=5.5.4
-  - conda-forge::pip=19.2.1
+  - bioconda::snakemake>=5.10
+  - pip=20.0.2
```

Comment:
Not sure if the conda-bot works without channel, why did you remove it?

Reply:
when installing the env, git complains that you didn't add pip as a dependency. This fixes that complaint.

Reply:
That's weird that this causes it... Seems like a bug on conda's side